src/style.css

```diff
@@ -316,6 +316,10 @@ button:focus {
   background-color: #888;
 }
 
+.bg-ssw-slate {
+  background-color: #777;
+}
```

Comment:
This should be in the tailwind config
https://github.com/SSWConsulting/SSW.Rules/blob/794a1f04c4d54c5398af0d076ec484b7f9e28ddd/tailwind.config.js#L18-L19